docusaurus/docs/develop/developer_guide/adding_params.md

```diff
@@ -43,245 +50,468 @@ https://github.com/ignite/cli/issues/3684#issuecomment-2299796210
+If the module doesn't already support this message, it will need to be added.
+
+#### 0.1 Scaffold the `MsgUpdateParam` Message
+
+Use `ignite` to scaffold a new `MsgUpdateParam` message for the module.
+Additional flags are used for convenience:
+
+```bash
+ignite scaffold message update-param --module examplemod --signer authority name as_type --response params
+```
+
+#### 0.2 Update the `MsgUpdateParam` Message Fields
+
+Update the `MsgUpdateParam` message fields in the module's `tx.proto` file to include the following comments and protobuf options:
+
+```protobuf
++ // MsgUpdateParam is the Msg/UpdateParam request type to update a single param.
+  message MsgUpdateParam {
+    option (cosmos.msg.v1.signer) = "authority";
+-   string authority = 1;
++ 
++   // authority is the address that controls the module (defaults to x/gov unless overwritten).
++   string authority = 1  [(cosmos_proto.scalar) = "cosmos.AddressString"];
++ 
+    string name      = 2;
+-   string asType    = 3;
++   oneof as_type {
++     // Add `as_<type>` fields for each type in this module's Params type; e.g.:
++     // int64 as_int64 = 3 [(gogoproto.jsontag) = "as_int64"];
++     // bytes as_bytes = 4 [(gogoproto.jsontag) = "as_bytes"];
++     // cosmos.base.v1beta1.Coin as_coin = 5 [(gogoproto.jsontag) = "as_coin"];
++   }
+  }
+  
+  message MsgUpdateParamResponse {
+```
+
+#### 0.3 Comment Out AutoCLI
+
+When scaffolding the `MsgUpdateParam` message, lines are added to `x/examplemod/module/autocli.go`.
+Since governance parameters aren't updated via `poktrolld` CLI, comment out these new lines:
+
+```go
+  // ...
+  Tx: &autocliv1.ServiceCommandDescriptor{
+      Service:              modulev1.Msg_ServiceDesc.ServiceName,
+      EnhanceCustomCommand: true, // only required if you want to use the custom command
+      RpcCommandOptions:    []*autocliv1.RpcCommandOptions{
+          // ...
++         //	{
++         //		RpcMethod:      "UpdateParam",
++         //		Use:            "update-param [name] [as-type]",
++         //		Short:          "Send a update-param tx",
++         //		PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "name"}, {ProtoField: "asType"}},
++         //	},
+          // this line is used by ignite scaffolding # autocli/tx
+      },
+  },
+  // ...
+```
+
+#### 0.4. Update the DAO Genesis Authorizations JSON File
+
+Add a grant (array element) to `tools/scripts/authz/dao_genesis_authorizations.json` with the `authorization.msg` typeURL for this module's `MsgUpdateType`:
+
+```json
++ {
++   "granter": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
++   "grantee": "pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw",
++   "authorization": {
++     "@type": "\/cosmos.authz.v1beta1.GenericAuthorization",
++     "msg": "\/poktroll.examplemod.MsgUpdateParam" // Replace examplemod with the module name
++   },
++   "expiration": "2500-01-01T00:00:00Z"
++ },
+```
+
```

Comment:
```suggestion

#### 0.5 Tidy Up

Run the module's unit tests to see where additional changes need to be made after updating the protobuf field types:

\```bash
go test ./x/examplemod/...
\```

```

Reply:
This supersedes the first half of section 8.0.

TODO: add another section (0.6) which supersedes the second half of section 8.0.